Healthy/Classes/Modules/Splash/DemoViewController.swift

```diff
@@ -0,0 +1,225 @@
+//
+//  OnboardingViewController.swift
+//  Healthy
+//
+//  Created by dodo on 15/05/2023.
+//
+
+
+import UIKit
+
+class DemoViewController: UIPageViewController {
+
+    var pages = [UIViewController]()
+    
+    let skipButton = UIButton()
+    let nextButton = UIButton()
+    let pageControl = UIPageControl()
+    let initialPage = 0
+
+    // animations
+    var pageControlBottomAnchor: NSLayoutConstraint?
+    var skipButtonTopAnchor: NSLayoutConstraint?
+    var nextButtonTopAnchor: NSLayoutConstraint?
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        setup()
+        style()
+        layout()
+    }
+}
+
+extension DemoViewController {
+    
+    func setup() {
```

Comment:
setup what?
You should explain the name of the function clearly
ex: `setupOnBoardingPagesView`